packages/actor-rdf-update-hypermedia-patch-sparql-update/lib/QuadDestinationPatchSparqlUpdate.ts

```diff
@@ -5,6 +5,7 @@ import { validateHttpResponse } from '@comunica/bus-rdf-update-quads';
 import type { IActionContext } from '@comunica/types';
 import type * as RDF from '@rdfjs/types';
 import type { AsyncIterator } from 'asynciterator';
+import { ArrayIterator, EmptyIterator } from 'asynciterator';
```

Comment:
`EmptyIterator` is not safe to use (with comunica's enforcement on `autoStart: false`).
Instead, we should always use `ArrayIterator([], { autoStart: false })`.
See https://github.com/RubenVerborgh/AsyncIterator/issues/25